code/modules/projectiles/guns/projectile/revolver/artwork_revolver.dm

```diff
@@ -31,9 +31,11 @@
 	damage_multiplier += pick(-0.2,-0.1,0,0.1,0.2)
 	penetration_multiplier += pick(-0.2,-0.1,0,0.1,0.2)
 	recoil_buildup += rand(-(recoil_buildup / 5),(recoil_buildup / 5))
-
+	price_tag += rand(0, 2500)
```

Comment:
Even a completely shitty toy gun is worth it's weight in scrap.
```suggestion
	price_tag += rand(200, 2500)
```

Reply:
the minimum price is one hundred and the maximum bonus is 2,500.